<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::ShareLinkWidget</class>
 <widget class="QWidget" name="OCC::ShareLinkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Share NewDocument.odt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="publicLinkWarningLabel">
     <property name="text">
      <string>Anyone with the link has access to the file/folder</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_create">
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="placeholderText">
        <string>Enter a name to create a new public link...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createShareButton">
       <property name="text">
        <string>&amp;Create new</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="linkShares">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="shareProperties" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_expire">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_expire">
          <property name="text">
           <string>Set &amp;expiration date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="calendar">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_setPassword">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Set &amp;password </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_password">
        <item>
         <widget class="QCheckBox" name="checkBox_password">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Set password</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Link properties:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QWidget" name="widget_editing" native="true">
        <layout class="QGridLayout" name="layout_editing">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_editing">
           <property name="text">
            <string>Allow editing</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_fileListing">
           <property name="text">
            <string>Show file listing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>123</red>
           <green>121</green>
           <blue>134</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>createShareButton</tabstop>
  <tabstop>linkShares</tabstop>
  <tabstop>checkBox_editing</tabstop>
  <tabstop>checkBox_password</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_setPassword</tabstop>
  <tabstop>checkBox_expire</tabstop>
  <tabstop>calendar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
